src/tiled/commandlineparser.cpp

```diff
@@ -72,8 +72,7 @@ bool CommandLineParser::parse(const QStringList &arguments)
         if (arg.length() == 1) {
             // Traditionally a single hyphen means read file from stdin,
             // write file to stdout. This isn't supported right now.
-            qWarning().nospace() << "Bad argument " << index
-                                 << ": lonely hyphen";
+            qWarning().nospace() << tr("Bad argument %1: lonely hyphen").arg(index);
```

Comment:
This will need to be wrapped with `qPrintable` to avoid it being quoted in the output. Also, the `nospace()` can be removed since there aren't multiple pieces streamed to the output anyway. So this should be:
```
qWarning() << qPrintable(tr("Bad argument %1: lonely hyphen").arg(index));
```
Other lines need to be changed similarly.